demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -375,8 +382,10 @@ int main(int argc, char *argv[]) {
             if (27 == key)  // Esc
                 break;
             if (9 == key) {  // Tab
-                isAsyncMode ^= true;
+                isAsyncMode = true;
```

Comment:
Doesn't this mean that async mode can no longer be reset?

Reply:
Reverted